### PR TITLE
sp/automatic-ml-intro-w-test-drive-es-translation

### PR DESCRIPTION
**Pull request by 5675sp**
Review Spanish translation of Outline, Objective, and Task 1.

### CODE REVIEW

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 1]
```diff
@@ -87,27 +87,27 @@ La interfaz del usuario (UI) de Driverless AI es muy fácil de navegar. Las sigu
...
 
 El aprendizaje supervisado toma los ejemplos etiquetados y permite a un modelo que esta siendo entrenado aprender la relación entre características y etiquetas. El modelo entrenado es entonces probado con datos no-etiquetados, y eso permite predecir el valor de y(etiqueta) para los datos no-etiquetados. Probar un modelo entrenado con datos no-etiquetados se le llama entrenamiento no supervisado [1]. Note que H20 Driverless AI crea modelos con ejemplos etiquetados.
 
 ### Preparación de datos
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:109]: Testing for review

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 2]
```diff
@@ -87,27 +87,27 @@ La interfaz del usuario (UI) de Driverless AI es muy fácil de navegar. Las sigu
 
 - [Aprende más sobre H2O Driverless al revisar nuestra lista de preguntas frecuentes](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/faq.html)
 
-## Tarea 2: Conceptos de Machine Learning(Aprendizaje de maquina)
+## Tarea 2: Conceptos de Machine Learning(Aprendizaje de Máquina)
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:90]: Agrega un espacio despues de learning: Conceptos de Machine Learning (Aprendizaje de Máquina)

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
 
 # Automatic Machine Learning Introduction with Driverless AI
 
-## Outline
+## Contorno
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:4]: Test- delete after

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 4]
```diff
@@ -87,27 +87,27 @@ La interfaz del usuario (UI) de Driverless AI es muy fácil de navegar. Las sigu
 
 - [Aprende más sobre H2O Driverless al revisar nuestra lista de preguntas frecuentes](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/faq.html)
 
-## Tarea 2: Conceptos de Machine Learning(Aprendizaje de maquina)
+## Tarea 2: Conceptos de Machine Learning(Aprendizaje de Máquina)
 
 ###  Inteligencia Artificial y Machine Learning
 
-Los conceptos que se encuentran en esta sección están destinados a proporcionar una vista general de alto nivel del Machine Learning(Aprendizaje de maquina). Al final de esta sección, podrás encontrar ligas(links) a recursos que ofrecen una explicación más profunda de los conceptos cubiertos aquí.
+Los conceptos que se encuentran en esta sección están destinados a proporcionar una vista general de alto nivel de Aprendizaje de maquina. Al final de esta sección, podrás encontrar ligas(links) a recursos que ofrecen una explicación más profunda de los conceptos cubiertos aquí.
```
Comment (eulisesv) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:94]: Capitalize the word "maquina" in Aprendizaje de Maquina

Comment (eulisesv) [reply Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:94]: Also, the word Maquina needs to have the tilde above the first "a"

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 5]
```diff
@@ -87,27 +87,27 @@ La interfaz del usuario (UI) de Driverless AI es muy fácil de navegar. Las sigu
...
+Avances en la tecnología han facilitado la recopilación y disponibilidad de los datos. Los tipos de datos disponibles determinarán el tipo de entrenamiento al que el modelo Aprendizaje de Maquina puede someterse. Hay dos tipos de entrenamiento de Aprendizaje de Maquina, Aprendizaje supervisado y no-supervisado. Aprendizaje supervisado es cuando el conjunto de datos contiene la salida que estás tratando de predecir. Para esos casos donde la variable de predicción no esta presente, se le llama aprendizaje no-supervisado. Ambos tipos de entrenamiento definen la relación entre variables de entrada y de salida.
```
Comment (eulisesv) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:100]: I don't think "salida" is the right word to use. I think "respuesta" might be better

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
+En esta descripción general de Driverless AI, aprenderá cómo cargar datos, explorar detalles de datos, generar visualizaciones automáticas, iniciar un experimento, explorar ingeniería de características, ver resultados de experimentos y obtener un recorrido rápido por el informe de Interpretabilidad de aprendizaje automático.
 
-**Note**: This tutorial has been built on Aquarium, which is H2O's cloud environment that provides access to various tools for workshops, conferences, and training. The labs in Aquarium have datasets, experiments, projects, and other content preloaded. If you use your version of Driverless AI, you will not see the preloaded content.
+**Nota**: Este tutorial se ha creado en Aquarium, que es el entorno de nube de H2O que proporciona acceso a varias herramientas para talleres, conferencias y capacitación. Los laboratorios en Aquarium tienen conjuntos de datos, experimentos, proyectos y otro contenido precargado. Si usa su versión de Driverless AI, no verá el contenido precargado.
```
Comment (eulisesv) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:27]: Not sure why Objective is being translated here. I took this as one of my actions, and I resubmitted my commit for the Objective section already. Let me know what needs to be done here.

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 1 above)
Comment (5675sp) [reply Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:109]: Just to make sure this was just a test, right? No review is required?

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 3 above)
Comment (5675sp) [reply Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:4]: Should I delete this comment? It seems it was for test purposes...

File: Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 7]
```diff
@@ -1,34 +1,34 @@
...
+- [Apéndice: Espacio de trabajo del proyecto](#appendix-project-workspace)
 
 
-## Objective
+## Objetivo
 
-For this tutorial, we will explore the Titanic dataset from the perspective of a passenger life insurance company with [H2O.ai's](https://www.h2o.ai/) enterprise product, [Driverless AI](https://www.h2o.ai/products/h2o-driverless-ai/). We will explore possible riks factors derived from this dataset that could have been considered when selling passenger insurance during this time. More specifically, we will create a predictive model to determine what factors contributed to a passenger surviving.
+Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [H2O.ai's](https://www.h2o.ai/), [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/). Exploraremos posibles factores de riesgo derivados de este conjunto de datos que podrían haberse considerado al vender el seguro de pasajeros durante este tiempo. Más específicamente, crearemos un modelo predictivo para determinar qué factores contribuyeron a la supervivencia de un pasajero.
```
Comment (Castro38) [inline Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md]: Suggestion to change: 
Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [H2O.ai's](https://www.h2o.ai/), [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/).

To:
Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/) de [H2O.ai's](https://www.h2o.ai/).

**Comment by 5675sp** [comment]
Second Tutorial: Translated Objective,Prerequisites, and Task 1: Launch Experiment.

**Comment by 5675sp** [comment]
Second Tutorial: Translated Task 2: Explore Experiment Settings and Expert Settings.

**Pull request merged**